config/api.yml

```diff
@@ -1122,6 +1122,11 @@
         :identifier: miq_ae_customization_explorer
       - :name: refresh_dialog_fields
         :identifier: svc_catalog_provision
+      - :name: delete
+        :identifier: dialog_delete
+      :delete:
+      - :name: delete
+        :identifier: dialog_delete
```

Comment:
The :delete: here should not be at the collection level (DELETE /api/service_dialogs).